### PR TITLE
add check for tls termination

### PR DESCRIPTION
**Pull request by Gl4di4torRr**
Enables re-encrypt and passthrough for tls termination. The default is edge.

### CODE REVIEW

**Comment by Gl4di4torRr** [comment]
@etsauer @Digi59404 let me know what you think.

File: pkg/stub/handler.go [hunk 1]
```diff
@@ -84,13 +84,23 @@ func (h *Handler) handleRoute(route *v1.Route) error {
...
-
 		var routeCopy *v1.Route
 		routeCopy = route.DeepCopy()
 		routeCopy.ObjectMeta.Annotations[h.config.General.Annotations.Status] = "no"
 		routeCopy.ObjectMeta.Annotations[h.config.General.Annotations.Expiry] = keyPair.Expiry.Format(timeFormat)
+
+		//var termination string
+
+		var termination v1.TLSTerminationType
+		config := route.Spec.TLS
```
Comment (Digi59404) [inline pkg/stub/handler.go]: Is `route.Spec` defined somewhere or is it dynamic. Just a list of objects in the spec yml?

**Comment by Digi59404** [comment]
@Gl4di4torRr  Did you test this already locally?

**Comment by Gl4di4torRr** [comment]
@Digi59404 Yes I tested this locally

File: pkg/stub/handler.go (hunk 1 above)
Comment (Gl4di4torRr) [reply pkg/stub/handler.go]: https://github.com/openshift/api/blob/master/route/v1/types.go#L67
maybe this will help answer your question

Comment (Digi59404) [reply pkg/stub/handler.go]: Yep! Looked at it with eric. He explained it. All good!

**Comment by etsauer** [comment]
testing...

**Pull request merged**